docs/guides/migrating-from-Camunda-Platform.md

```diff
@@ -0,0 +1,400 @@
+* ```camunda:failedJobRetryTimeCycle```: You cannot yet configure the retry time cycle.
+
+### Message receive events and receive tasks
+
+Message correlation works slightly different between the two products:
+
+* Camunda Platform simply waits for a message, and the code implementing that the message is received queries for a process instance the message will be correlated to. If no process instance is ready to receive that message, an exception is raised.
+
+* Camunda Cloud creates a message subscription for every waiting process instance. This subscription requires a value for a ```correlationKey``` to be generated when entering the receive task. The code receiving the external message correlates using the value of the ```correlationKey```.
+
+This means you must inspect and adjust all message receive events or receive tasks in your model to define a reasonable ```correlationKey```. You also must adjust your client code accordingly.
+
+The ```bpmn message name``` is used in both products and doesn't need migration. 
+
+## Prepare for smooth migrations
+
+Whenever you build a process solution using Camunda Platform, you can follow these recommendations to create a process solution that will be easier to migrate later on:
+
+* Use Java, Maven, and Spring Boot.
+* Separate your business logic from Camunda API.
+* Use external tasks.
+* Stick to basic usage of public API (no engine plugins or extensions).
+* Don't expose Camunda APIs (REST or Java) to front-end applications.
+* Use primitive variable types or JSON payloads only (no XML or serialized Java objects).
+* Use JSONPath on JSON payloads (translates easier to FEEL).
+* Stick to [BPMN elements supported in Camunda Cloud](https://docs.camunda.io/docs/reference/bpmn-processes/bpmn-coverage).
+* Use [FEEL as script language in BPMN](https://camunda.github.io/feel-scala/docs/reference/developer-guide/bootstrapping#use-as-script-engine), e.g. on Gateways.
+* Use Camunda Forms.
+
+## Open issues
+
+As described earlier in this guide, migration is an ongoing topic and this guide is far from complete. Open issues include the following:
+
+* Describe implications on testing.
+* Discuss adapters for Java or REST client.
+* Discuss external task adapter for Java code and probably add it to the [Camunda Platform to Cloud Adapter](https://github.com/berndruecker/camunda-platform-to-cloud-migration/tree/main/camunda-platform-to-cloud-adapter).
+* Discuss more concepts around BPMN
+** [Field injection](https://docs.camunda.org/manual/latest/user-guide/process-engine/delegation-code/#field-injection) that is using ```camunda:field``` available on many BPMN elements.
+** Multiple instance markers available on most BPMN elements.
+** ```camunda:inputOutput``` available on most BPMN elements.
+** ```camunda:errorEventDefinition``` available on several BPMN elements.
+
+And even more. 
+
+Please [reach out to us](https://camunda.com/contact/) to discuss your specific migration use case.
```

Comment:
Same comment here, would this contact page make more sense - https://docs.camunda.io/contact ?

Reply:
https://github.com/berndruecker/camunda-cloud-documentation/commit/fc4bc097ce039eb722cbf6f328277bdd08119658